api-docs/rst/dev-guide/api-operations/methods/get-get-scaling-group-state-v1.0-tenantid-groups-groupid-state.rst

```diff
@@ -14,14 +14,17 @@ This operation retrieves the current state of a scaling group.
 The *GroupState* object consists of the following properties:
 
 
-
-*  *paused*. Specifies whether execution of scaling policies for the group is currently suspended. If this value is set to true, the group will not scale up or down. All policy execution calls are ignored while this value is set to true.
+*  *paused*. Specifies whether execution of scaling policies for the group is currently suspended. If this value
+   is set to TRUE, the group will not scale up or down. All scheduled and API-generated policy executions are suspended, and convergence will
+   not be triggered while this value is set to TRUE. Any *POST* calls to the :ref:`/converge <post-create-scaling-group-v1.1-tenantid-converge>` operation and any *POST* calls to the :ref:`/execute <post-execute-policy-v1.0-tenantid-groups-groupid-policies-policyid-execute>`
+   operation will error with a `403GroupPausedError` error message if convergence is paused. If this value is set FALSE, all scaling and convergence activities, as well as all policy execution calls are resumed.
```

Comment:
I found the original text pretty confusing, which is probably part of the reason my revision was not quite accurate.  I realize the mistake I made was in the reference -- I looked at the name "POST create scaling group", not reading it as the converge endpoint. What I was trying to accomplish was to separate what happens when the value is true and what happens when it is false. 

If 



As far as run vs. execute -- it's a style choice. Search for "execute" in [Rackspace style guide](https://github.com/rackerlabs/docs-rackspace/blob/efaf97afced7c799d886dec3af8f1e623db6231d/style-guide/basic-writing-guidelines.md)

```
Execute policy

POST /v1.0/{tenantId}/groups/{groupId}/policies/{policyId}/execute

This operation **runs** a specified scaling policy.
```

I'll update my revised version in the comment above.